### PR TITLE
DOCS: Add meat of  MPP-ized system views and note which summary views the relevant views are summarized in

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**
Staging here: https://docs-staging.vmware.com/en/Greenplum-Database-7/GPDOCS-578-system-views-phase2/greenplum-database/GUID-ref_guide-system_catalogs-catalog_ref-views.html

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 1]
```diff
@@ -727,6 +1160,22 @@ The `pg_stat_user_tables` and `pg_stat_sys_table` views contain the same informa
...
+|------|----|----------|-----------|
+|`classname`|text| |The name of the system table in the `pg_catalog` schema where the record about this object is stored \(`pg_class`=relations, `pg_database`=databases,`pg_namespace`=schemas, `pg_authid`=roles\)|
+|`objname`|name| |The name of the object.|
+|`objid`|oid| |The OID of the object.|
+|`schemaname`|name| |The name of the schema where the object resides.|
+|`usestatus`|text| |The status of the role who performed the last operation on the object \(`CURRENT`=a currently active role in the system, `DROPPED`=a role that no longer exists in the system, `CHANGED`=a role name that exists in the system, but has changed since the last operation was performed\).|
+|`usename`|name| |The name of the role that performed the operation on this object.|
+|`actionname`|name| |The action that was taken on the object.|
+|`subtype`|text| |The type of object operated on or the subclass of operation performed.|
+|`statime`|timestamptz| |The timestamp of the operation. This is the same timestamp that is written to the Greenplum Database server log files in case you need to look up more detailed information about the operation in the logs.|
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:1177]: This view currently also exists in "[System Catalogs Definitions](https://docs-staging.vmware.com/en/Greenplum-Database-7/GPDOCS-578-system-views-phase2/greenplum-database/GUID-ref_guide-system_catalogs-catalog_ref-html.html)". Should we delete it there?

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 2]
```diff
@@ -320,186 +448,184 @@ Status for specific tables involved in the expansion is stored in [gpexpand.stat
...
+|`seq_tup_read`|bigint| |Number of live rows fetched by sequential scans.|
+|`idx_scan`|bigint| |Number of index scans initiated on this table.|
+|`idx_tup_fetch`|bigint| |Number of live rows fetched by index scans.|
+|`n_tup_ins`|bigint| |Number of rows inserted.|
+`n_tup_upd`|bigint| |Number of rows updated (includes HOT updated rows).|
+|`n_tup_del`|bigint| |Number of rows deleted.|
+|`n_tup_hot_upd`|bigint| |Number of rows HOT updated (that is, with no separate index update required).|
+|`n_live_tup`|bigint| |Estimated number of live rows.|
+|`n_dead_tup`|bigint| |Estimated number of dead rows.|
+|`n_mod_since_analyze`|bigint|Estimated number of rows modified since this table was last analyzed.|
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:513]: format is off, should be:
```
|`n_mod_since_analyze`|bigint| |Estimated number of rows modified since this table was last analyzed.|
```

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 3]
```diff
@@ -320,186 +448,184 @@ Status for specific tables involved in the expansion is stored in [gpexpand.stat
...
+|`analyze_count`|bigint| |Number of times this table has been manually analyzed.|
+|`autoanalyze_count`|bigint| |Number of times this table has been analyzed by the autovacuum daemon.|
+
+This system view is summarized in the `gp_stat_all_tables_summary` system view.
+
+## <a id="gp_stat_archiver"></a>gp_stat_archiver
+
+|column|type|references|description|
+|------|----|----------|-----------|
+|`gp_segment_id`|integer| |Unique identifier of a segment (or coordinator) instance.||`archived_count`|bigint| |Number of WAL files that have been successfully archived.|
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:530]: format seems off

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 4]
```diff
@@ -320,186 +448,184 @@ Status for specific tables involved in the expansion is stored in [gpexpand.stat
...
+|`deadlocks`|bigint| |Number of deadlocks detected in this database.|
+|`checksum_failures`|bigint| |Number of data page checksum failures detected in this database (or on a shared object), or NULL if data checksums are not enabled.|
+|`checksum_last_failure`|timestamp with time zone| |Time at which the last data page checksum failure was detected in this database (or on a shared object), or NULL if data checksums are not enabled.|
+|`blk_read_time`|double precision| |Time spent reading data file blocks by backends in this database, in milliseconds (if track_io_timing is enabled, otherwise zero).|
+|`session_time`|double precision| |Time spent by database sessions in this database, in milliseconds (note that statistics are only updated when the state of a session changes, so if sessions have been idle for a long time, this idle time won't be included).|
+|`active_time`|double precision| Time spent executing SQL statements in this database, in milliseconds (this corresponds to the states active and fastpath function call in pg_stat_activity).|
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:583]: format seems off - need a blank column between reference and description

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 5]
```diff
@@ -320,186 +448,184 @@ Status for specific tables involved in the expansion is stored in [gpexpand.stat
...
+|`sessions_abandoned`|bigint| |Number of database sessions to this database that were terminated because connection to the client was lost.|
+|`sessions_fatal`|bigint| |Number of database sessions to this database that were terminated by fatal errors.|
+|`sessions_killed`|bigint| |Number of database sessions to this database that were terminated by operator intervention.|
+|`stats_reset`|timestamp with time zone| |Time at which these statistics were last reset.|
+
+This system view is summarized in the `gp_stat_database_summary` system view.
+
+## <a id="gp_stat_database_conflicts"></a>gp_stat_database_conflicts
+
+|`gp_segment_id`|integer| |Unique identifier of a segment (or coordinator) instance.|
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:595]: Need header:
```
|column|type|references|description|
|------|----|----------|-----------|
```

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 6]
```diff
@@ -320,186 +448,184 @@ Status for specific tables involved in the expansion is stored in [gpexpand.stat
...
+|`confl_tablespace`|bigint| |Number of queries in this database that have been canceled due to dropped tablespaces.|
+|`confl_lock`|bigint| |Number of queries in this database that have been canceled due to lock timeouts.|
+|`confl_snapshot`|bigint| |Number of queries in this database that have been canceled due to old snapshots.|
+|`client_port`|integer| |Client port number.|
+|`confl_bufferpin`|bigint| |Number of queries in this database that have been canceled due to pinned buffers.|
+|`confl_deadlock`|bigint| |Number of queries in this database that have been canceled due to deadlocks.|
+
+## <a id="gp_stat_gssapi"></a>gp_stat_gssapi
+
+|`gp_segment_id`|integer| |Unique identifier of a segment (or coordinator) instance.|
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:607]: Need header:
```
|column|type|references|description|
|------|----|----------|-----------|
```

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 7]
```diff
@@ -526,6 +652,310 @@ The `gp_stat_replication` view contains replication statistics of the `walsender
...
+
+## <a id="gp_stat_ssl"></a>gp_stat_ssl
+
+The `gp_stat_ssl` view is a cluster-wide view that displays the `pg_stat_ssl` information from every primary segment.
+
+|column|type|references|description|
+|------|----|----------|-----------|
+|`gp_segment_id`|integer| |Unique identifier of a segment (or coordinator) instance.|
+|`pid`|integer|Process ID of a backend or WAL sender process.|
+|`ssl`|boolean|True if SSL is used on this connection|
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:696]: format seems off - descriptions are in the references column

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 8]
```diff
@@ -846,34 +1295,4 @@ The `pg_stats` view provides access to the information stored in the `pg_statist
...
-- gp_stat_xact_user_tables_summary
-- gp_statio_all_indexes_summary
-- gp_statio_all_sequences_summary
-- gp_statio_all_tables_summary
-- gp_statio_sys_indexes_summary
-- gp_statio_sys_sequences_summary
-- gp_statio_sys_tables_summary
-- gp_statio_user_indexes_summary
-- gp_statio_user_sequences_summary
-- gp_statio_user_tables_summary
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:877]: Will these summary views be brought back in a separate PR?

**Review by huansong — COMMENTED** [review]
Thanks for the LOTs of work! Mostly just formatting issues. I didn't finish checking all of the tables, could you please check and make sure the tables are displayed properly?

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md (hunk 1 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:1177]: Good question. I think the System Catalogs Definitions topic needs to be revisited. It should be a list of system tables only now, no system views. But then the title doesn't make sense. I'll have to check with my GPDB writing teammates.

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md (hunk 8 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:877]: Yes.
